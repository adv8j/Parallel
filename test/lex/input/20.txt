@parallel(shared=[sum], private=[c, d], num_threads=4, schedule=static)
for c in 0..100 {
    sum += someArray(a[d]);
}
