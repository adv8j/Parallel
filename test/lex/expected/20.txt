PARALLEL
LPAREN
SHARED
ASSIGN
LBRACKET
IDENTIFIER
RBRACKET
COMMA
PRIVATE
ASSIGN
LBRACKET
IDENTIFIER
COMMA
IDENTIFIER
RBRACKET
COMMA
IDENTIFIER
ASSIGN
NUMBER
COMMA
SCHEDULE
ASSIGN
STATIC_SCHEDULE
RPAREN
FOR
IDENTIFIER
IN
NUMBER
RANGE
NUMBER
LBRACE
IDENTIFIER
ADD_ASSIGN
IDENTIFIER
LPAREN
IDENTIFIER
LBRACKET
IDENTIFIER
RBRACKET
RPAREN
SEMICOLON
RBRACE
